src/shared/test/java/com/couchbase/lite/LogTest.java

```diff
@@ -298,6 +310,38 @@ public void testFileLoggingHeader() throws Exception {
             });
     }
 
+    @Test
+    public void testBasicLogFormatting() {
+        BasicLogger logger = new BasicLogger();
+        Database.log.setCustom(logger);
+
+        Log.d(LogDomain.DATABASE, "TEST DEBUG");
+        assertEquals("TEST DEBUG", logger.getMessage());
+
+        Log.d(LogDomain.DATABASE, "TEST DEBUG", new Exception("whoops"));
+        assertEquals("TEST DEBUG (java.lang.Exception: whoops)", logger.getMessage());
+
+        Log.d(LogDomain.DATABASE, "TEST DEBUG %s %d %f", "arg", 1, 3.0F);
+        assertEquals("TEST DEBUG arg 1 3.000000", logger.getMessage());
```

Comment:
is it always sure to have exactly 6 decimal places in all places?

Reply:
Yeah.... I can fix that with format `%.2f`.  Will do.